{"version": "0.5", "lang": "python", "startTime": "1583660353863", "uuid": "1aac9e2a-11a0-46bc-81ea-c7dddc1888f3", "newcontainer": "1", "vmuptime": "1583656233", "platform": "AWS Lambda", "containerID": "2020/03/08/[$LATEST]5fdb7bbf73074f788305476c74717303", "functionName": "ServiceTwoPython", "functionMemory": "3008", "functionRegion": "us-east-1", "vmID": "cuQZ6b", "linuxVersion": "#1 SMP Tue Aug 20 23:10:42 UTC 2019", "totalMemory": "3230668", "freeMemory": "3074060", "pageFaults": "71266", "cpuType": "Intel(R) Xeon(R) Processor @ 2.50GHz - Model 62", "cpuModel": "62", "cpuUsr": "63", "cpuNice": "0", "cpuKrn": "100", "cpuIdle": "823875", "cpuIowait": "3", "cpuIrq": "0", "cpuSoftIrq": "2", "vmcpusteal": "45", "contextSwitches": "25030", "frameworkRuntime": "30", "endpoint": "admin28.cluster-cftyxgwyrild.us-east-1.rds.amazonaws.com", "bucketname": "project.bucket.tcss562.sps", "key": "edited_100000_Sales_Records.csv", "error2": "[1041; 'Database was unable to resume within timeout period.']", "userRuntime": "91561", "cpuUsrDelta": "34", "cpuNiceDelta": "0", "cpuKrnDelta": "30", "cpuIdleDelta": "18240", "cpuIowaitDelta": "0", "cpuIrqDelta": "0", "cpuSoftIrqDelta": "0", "vmcpustealDelta": "2", "contextSwitchesDelta": "4290", "pageFaultsDelta": "56495", "frameworkRuntimeDeltas": "14", "runtime": "91605", "endTime": "1583660445468", "2_thread_id": "27", "1_run_id": "0", "zAll": "Final Results:", "roundTripTime": "92981.06", "payload": "{\"bucketname\": \"project.bucket.tcss562.sps\"; \"key\": \"edited_100000_Sales_Records.csv\"; \"tablename\": \"SalesData\"; \"batchSize\": 1000; \"dbEndpoint\": \"admin28.cluster-cftyxgwyrild.us-east-1.rds.amazonaws.com\"; \"dbName\": \"DB_TCSS562\"}", "latency": "1376.06", "runtimeOverlap": "19.97"}